cellpack/autopack/loaders/recipe_loader.py

```diff
@@ -116,10 +116,30 @@ def _request_sub_recipe(self, inode):
             return None
         return data
 
-    def _migrate_version(self, recipe, format_version="1.0"):
+    @staticmethod
+    def _sanitize_format_version(recipe_data):
+        if "format_version" not in recipe_data:
+            format_version = "1.0"  # all recipes before we introduced versioning
+        elif len(recipe_data["format_version"].split(".")) > 2:
+            # We only use two places for format version, but people
+            # might accidently include a third number
+            # ie 2.0.0 instead of 2.0
+            split_numbers = recipe_data["format_version"].split(".")
+            format_version = f"{split_numbers[0]}.{split_numbers[1]}"
+        elif len(recipe_data["format_version"].split(".")) == 1:
+            # We only use two places for format version, but people
+            # might accidently include a third number
+            # ie 2.0.0 instead of 2.0
```

Comment:
This comment is copied from the section above I believe? and does not apply to this section, which is about adding a point number if only a single number is provided.